Update toolbar actions

kisantia opened the pull request:
- Remove edit from home toolbar
- Add Refresh to the toolbar of other pages too if it has widgets container, grid container, or nav section because these have refresh implemented. Refresh for other tab content types are no ops so refresh won't be shown there because the extensions should define the refresh action themselves.

![image](https://user-images.githubusercontent.com/31145923/75394904-0a5e7780-58a6-11ea-802a-8af40e18c271.png)

![image](https://user-images.githubusercontent.com/31145923/75205931-fd1c7e00-5729-11ea-899a-d35a56b96ba1.png)

![image](https://user-images.githubusercontent.com/31145923/75291283-c486ae00-57d6-11ea-8599-787d2f34d45b.png)

Charles-Gagnon commented on src/sql/workbench/contrib/dashboard/browser/core/dashboardPage.component.ts:
```diff
@@ -396,6 +395,9 @@ export abstract class DashboardPage extends AngularDisposable implements IConfig
 			if (index !== -1) {
 				this.tabToolbarActionsConfig.set(value.id, configs[index].widget['tasks-widget']);
 				const content = this.getToolbarContent(configs[index].widget['tasks-widget']);
+				if (content.length > 0) {
```
Why only if there's other actions? I thought you were going to have refresh always show?

kisantia replied:
Discussed this with Alan and changed this so that refresh will show in the toolbar if the tab content is a widgets container, grid container, or nav section because those have refresh implemented and will refresh the widgets. If an extension is using something else for the content, like modelview container, they should define their own refresh.

kisantia commented:
@Charles-Gagnon Yes makes sense. Changed refresh scope to only be for the current page and added refresh to the toolbar for tabs with widgets even if there aren't other actions contributed to the toolbar.
